public/index.html

```diff
@@ -18,20 +19,42 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Spotify Clone</title>
   </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
+  <body class="app">
+    <!-- <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div> -->
+    <header class="header">
+      <a href="" class="header__logo" href="/">
+        <img src="./logo.png" alt ="logo" width="32" height="32" />
```

Comment:
Хорошая практика придерживаться единого стиля написания кода
Сейчас есть лишний пробел
```suggestion
        <img src="./logo.png" alt="logo" width="32" height="32" />
```